dev-3/dev-3/Transliterator.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace dev_3
+{
+    /// <summary>
+    /// Class for transliting sentences.
+    /// Fields may look strange, because methods are working with indices of arrays.
+    /// All transliterations are made according to transliteration table at https://www.westernunion.ru/ru/ru/transliteration-table.html
+    /// </summary>
+    public class Transliterator
+    {
+        public string Sentence { get; }
+        private bool IsRussian = false;
```

Comment:
use private static readonly string[] not to create massive for any object of Transliterator class